### ISSUE TITLE
AICH, UP and OH datasets

### ISSUE THREAD

**Issue by tiangexiang** [Arthur151/ROMP#75]
Thanks for your outstanding work and repo!

In your paper, you have noticed that AICH, UP and OH datasets are also included during training (and the dataloaders are provided). However, it seems that they are missing in the data preparation guide and cannot be downloaded. 

Is it possible that you can provide these data as well? 

Thanks again!

**Comment by Arthur151** [comment]
Thanks for your reminding. The training code is released to re-implement our ICCV paper. In the latest ICCV paper, we have dropped the usage of redundant datasets for comparison. But for your convenience of training different datasets, we release the related code. I just upload the latest version of ROMP to ArXiv. If you are interested in these datasets, you can get access to them via these links, [3DOH50K](https://seueducn1-my.sharepoint.com/personal/yangangwang_seu_edu_cn/_layouts/15/onedrive.aspx?id=%2Fpersonal%2Fyangangwang%5Fseu%5Fedu%5Fcn%2FDocuments%2F3DOH50K&originalPath=aHR0cHM6Ly9zZXVlZHVjbjEtbXkuc2hhcmVwb2ludC5jb20vOmY6L2cvcGVyc29uYWwveWFuZ2FuZ3dhbmdfc2V1X2VkdV9jbi9FczZrRnZHTzJXMUdxRDJnbzVQQUF5MEItZlcxZUprVWxYQzVCTHA5dGd1VGpnP3J0aW1lPXRDUmhQQUY1MlVn), [UP](https://files.is.tuebingen.mpg.de/classner/up/#datasets).

**Comment by tiangexiang** [comment]
Thanks for your response! Just double check, in your ICCV paper:

1) The only datasets that have SMPL param gts you used are **h36m** and **MPII**? 
2) Did your use the 3DPW val set or the test set in Table 1 (protocol 1)?
3) Also, in Table 2 (protocol 2), did you use any gts during inference?

Thanks for your time!!

**Comment by Arthur151** [comment]
1. The "SMPL param gts" come from Human3.6M. MPI-INF-3DHP doesn't have the SMPL parameters provided. 
2. The evaluation protocol 1 (Tab. 1) uses the entire 3DPW dataset.
3. ROMP takes the entire image are input (not the cropped image patches). We don't need any gts during inference.

**Comment by tiangexiang** [comment]
Hi, in your Tab. 1 you compared directly to BMP. However, in the BMP paper, they claimed they only used 3DPW testing set for evaluation (Tab. 5), but you used the entire 3DPW dataset (train+val+testing) for evaluation?
Can you please confirm this, please? Thanks!

**Comment by Arthur151** [comment]
Hi, Tiange,

Thank you for your concern


BMP claims that they follow the evaluation protocol of VIBE on 3DPW. But all results on Tab. 5 they reported are so weird. As far as I know, VIBE doesn't have a evaluation protocol like that. 


They haven't released the code. I will update the correct results on arxiv when they release the code Otherwise, I will delete it.



Get Outlook for Android
